tck/rest/src/main/java/org/eclipse/microprofile/metrics/test/MpMetricTest.java

```diff
@@ -745,10 +667,7 @@ public void testApplicationHistogramUnitBytesOpenMetrics() {
                 .body(containsString(prefix + "bytes_count"))
                 .body(containsString(prefix + "bytes_sum"))
                 .body(containsString("# TYPE application_" + prefix + "bytes summary"))
-                .body(containsString(prefix + "mean_bytes"))
-                .body(containsString(prefix + "min_bytes"))
-                .body(containsString(prefix + "max_bytes"))
```

Comment:
test for max should not be removed

Reply:
the "bytes_max"  has been included below as micrometer lists the unit first then the suffix.